uint8 brake_light
uint8 indicator
